modules/vec2.lua

```diff
@@ -321,6 +321,35 @@ function vec2.to_polar(a)
 	return radius, theta
 end
 
+-- Convert vec2 to vec3.
+-- @tparam vec2 a Vector to convert.
+-- @tparam number the new z component.
+-- @treturn vec3 Converted vector
+function vec2.append(a, z)
```

Comment:
I feel like this should explicitly be something like to_vec3(a, z), append suggests happening in place